fem/fespace.cpp

```diff
@@ -2091,19 +2089,27 @@ SparseMatrix* FiniteElementSpace::DerefinementMatrix(int old_ndofs,
             int r = DofToVDof(dofs[i], vd);
             int m = (r >= 0) ? r : (-1 - r);
 
-            if (!mark[m])
+            if (is_dg || !mark[m])
             {
                lR.GetRow(i, row);
-               R->SetRow(r, old_vdofs, row);
+
+               for (int j = 0; j < old_vdofs.Size(); j++)
```

Comment:
How is this different than using `SetRow` ?

Reply:
The complexity of `SetRow` is linear in the number of nonzeroes in the row. The complexity of the loop using `Set(int,int,double)` is quadratic, so `SetRow` should be preferred.

Reply:
@rw-anderson is there a reason while `SetRow` was replaced with the explicit loop? Can we revert?